ViewModel/Inventory/DefineSIAttributesRowItem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using HEC.CS.Collections;
+
+namespace ViewModel.Inventory
+{
+    public class DefineSIAttributesRowItem
+    {
+        public string Name { get; set; }
+
+        /// <summary>
+        /// The user defined default value
+        /// </summary>
+        public CustomObservableCollection<string> Items { get;} = new CustomObservableCollection<string>();
+        /// <summary>
+        /// The selected value from the combobox
+        /// </summary>
+        public string SelectedItem { get; set; }
+
+        public DefineSIAttributesRowItem(string name)
+        {
+            Name = name;
+        }
+
+        public bool IsValid()
+        {
+            return SelectedItem != null && SelectedItem != "";
```

Comment:
do !string.isNullOrEmpty(SelectedItem);

Reply:
oh, cool. I like that. Not sure i have every used that method before. Good to know.